src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40;
+  const discount = 20;
+  const discountMax = 50;
+  const sum = days * rentalCost;
+
+  if (days >= 7) {
+    return sum - discountMax;
+  } else if (days >= 3) {
+    return sum - discount;
+  } else {
+    return sum;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 3rd item, you don't need else's here
```suggestion
  } 
  
  if (days >= 3) {
    return sum - discount;
  }
  
  return sum;
```